Description/fr-FR.md

```diff
@@ -0,0 +1,83 @@
+# Français (FR)
+
+Un modpack basé sur Fabric pour Minecraft orienté sur l'optimisation graphique et des performances. 
+
+[![Current review](https://img.youtube.com/vi/bb8G9X5Q_4I/maxresdefault.jpg)](https://www.youtube.com/watch?v=bb8G9X5Q_4I)
```

Comment:
You forgot the alt text. If you can find a French review, you can replace the link as well.